tests/beaker/tests/create_vg_param_createonly.rb

```diff
@@ -0,0 +1,47 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4579 - C96596 - create volume group with parameter 'createonly'"
```

Comment:
Nice catch!
Modified the test to create a volume group, and then attempt to create the same-name volume group with 'createonly' and make sure the original volume group is unchanged